docker/Dockerfile

```diff
@@ -15,14 +15,12 @@
 # limitations under the License.
 #
 
-# Usage:
-#   1. use ./build/dist to make binary distributions of Kyuubi or download a release
-#   2. Untar it and run the docker command below
-#      docker build -f docker/Dockerfile -t repository/kyuubi:tagname .
-#   Options:
-#     -f this docker file
-#     -t the target repo and tag name
-#     more options can be found with -h
+# It's recommended to use ./bin/docker-image-tool.sh to build the image rather than the vanilla docker command.
```

Comment:
cc @zwangsheng, am I right?

Reply:
Yes, although we may need to optimize this script for now, we still recommend that users use script builds to reduce the complexity of building docker images.